data/human/human missions.txt

```diff
@@ -7410,3 +7410,738 @@ mission "Adelita 5"
+		conversation
+			`You arrive at the address Ildico gave you; a large warehouse on the edge of the spaceport. When you step into the dimly lit building, your eyes struggle to adjust, but when you're able to see clearly you find yourself face-to-face with a group of dirty burly men and women leveling a half-dozen weapons right at you.`
+			`	"Looks like someone's lost," a voice rises from the group, though you can't tell who's speaking.`
+			`	"They look right fancy. Good thing we found them."`
+			`	"Whaddya say, Frank, kill 'em now? Or keep 'em for a bit?"`
+			choice
+				`	"Ildy?"`
+				`	"Ildy asked me to come here."`
+				`	"Ildy, help!"`
+				`	(Grab your sidearm and shoot your way out.)`
+					goto why?
+
+			`	The band of cut-throats looks at you, confused, and then most of them lower their weapons.`
+			`	"Boss," one of them hollers, "this stranger says they're here for you, I think."`
+			`	Footsteps echo on the opposite side of the warehouse as someone approaches.`
+			`	One of the men starts laughing. "Ildy?! Haha, never thought to call her that. Hey Ildy! How come you never told us to call you that? Or that some kind of special pet name?"`
+			`	The man flashes an unsavory grin at you. His rotting teeth on full display.`
+			`	In the dim light the footsteps materialize into a woman in her mid-thirties with hair darker than vacuum. Ildico Drifa gives you a warm smile, she then raises up her hand and fires her side-arm.`
+			choice
+				`	(Duck.)`
+
+			`	You duck, startled by the attack, only to realize she wasn't aiming at you. The rotten-toothed man lay dead on the ground, a bullet hole in his head.`
+			`	She looks to the rest of her crew, "Only my friends call me Ildy. And I have only one friend."`
+			`	"Yes, boss."`
+			`	"Got it, boss."`
+			`	"Sure thing, Cap."`
+			`	"Whatever you say, boss."`
+			`	The group of pirates melt away into the warehouse, one of them dragging the dead body, leaving you and Ildy relatively alone.`
+			`	"I'm so glad you came, <name>." Ildico puts a friendly hand on your shoulder and squeezes.`
+			`	She turns and waves toward the warehouse. "This is my temporary headquarters. I have few other safe-houses, and own part of a hangar elsewhere. Not too many ships yet, but I intend to change that soon."`
+			`	Ildico holsters her gun right next to where she keeps her dagger and continues. "But making moves is hard. Everyone wants to be on top. Which is why I need you."`
+			`	She looks you in the eye and says, "There's a rival gang right next door to us. They have a lot more firepower, but if they could be removed, we could easily double our territory.`
+			`	"I don't have the ships," she glances at her gang in the distance, "or the quality of crew, to deal with them."`
+			`	"But, I think you do. I can't afford to pay you right now, things are pretty fluid, but I'll owe you. What do you say?"`
+			choice
+				`	"Sure, I'll help."`
+				`	"Fine. But you're going to owe big when this is all over."`
```

Comment:
\<first\> \<last\> still isn’t getting paid?

Reply:
Not everything is a payday ;)